examples/online/index.js

```diff
@@ -4,15 +4,15 @@
 // https://redis.io/
 
 // then:
-// $ npm install redis online
+// $ npm install redis
 // $ redis-server
 
 /**
  * Module dependencies.
  */
 
 var express = require('../..');
-var online = require('online');
+var online = require('./online-tracker');
```

Comment:
Just remove this entire file and pull the logic into here. It is an example, and examples this small should be easy to follow in a single file.